### ISSUE TITLE
Bugsnag for Search

### ISSUE THREAD

**Issue by mfenner** [datacite/datacite#280]
Add Bugsnag error tracking to Search.

**Comment by kjgarza** [comment]
BLOCKER:

Container cannot be build at the moment due to:

```
Downloading Oracle Java 8...
--2018-01-19 13:00:00--  http://download.oracle.com/otn-pub/java/jdk/8u151-b12/e758a0de34e24606bca991d704f6dcbf/jdk-8u151-linux-x64.tar.gz
Resolving download.oracle.com (download.oracle.com)... 104.104.142.192
Connecting to download.oracle.com (download.oracle.com)|104.104.142.192|:80... connected.
HTTP request sent, awaiting response... 302 Moved Temporarily
Location: https://edelivery.oracle.com/otn-pub/java/jdk/8u151-b12/e758a0de34e24606bca991d704f6dcbf/jdk-8u151-linux-x64.tar.gz [following]
--2018-01-19 13:00:00--  https://edelivery.oracle.com/otn-pub/java/jdk/8u151-b12/e758a0de34e24606bca991d704f6dcbf/jdk-8u151-linux-x64.tar.gz
Resolving edelivery.oracle.com (edelivery.oracle.com)... 104.92.104.136, 2a02:26f0:cf:282::2d3e, 2a02:26f0:cf:28c::2d3e
Connecting to edelivery.oracle.com (edelivery.oracle.com)|104.92.104.136|:443... connected.
HTTP request sent, awaiting response... 302 Moved Temporarily

```
Try another day.

**Comment by kjgarza** [comment]
combine with change in solr template need to add bugsnag variable there.

**Comment by mfenner** [comment]
Implemented via datacite/search#162.
